src/main/routes/jurisdiction.ts

```diff
@@ -10,6 +10,7 @@ router.get("/jurisdiction", (req, res, next) => {
     res.status(200);
     const responseContent: { [k: string]: any } = {};
     responseContent.adminWebAuthorization = req.adminWebAuthorization;
+    responseContent.user = JSON.stringify(req.authentication.user);
```

Comment:
can this be unit tested?

Reply:
@gbenadikar - I've amended the tests to check for the presence of the user data within a hidden `input` on the HTML page